scripts/upstream_files.txt

```diff
@@ -1,9 +1,18 @@
+accounts/*
+consensus/*
 core/*
 eth/*
+log/*
+miner/*
 node/*
 internal/*
```

Comment:
These look like they are supposed to be sorted, can we make sure they are all sorted as expected?

Reply:
Yeah I also like it being sorted.